ui/apps/platform/src/Containers/ExceptionConfiguration/ExceptionConfigurationPage.tsx

```diff
@@ -5,16 +5,16 @@ import PageTitle from 'Components/PageTitle';
 import useURLStringUnion from 'hooks/useURLStringUnion';
 import VulnerabilitiesConfiguration from './VulnerabilitiesConfiguration';
 
-const deferralConfigurationCategories = ['Vulnerabilities'] as const;
+const exceptionConfigurationCategories = ['Vulnerabilities'] as const;
```

Comment:
David, as a learning opportunity, can you remind me in which ways this helps TypeScript help us?

I suspect that my work in progress can benefit, but do not want to add without understanding why.

Reply:
This helps TypeScript infer that `exceptionConfigurationCategories` is the most narrow, read-only type possible. Since the array is declared as read-only, the type of `exceptionConfigurationCategories` is `['Vulnerabilities']` instead of `string[]`.

This allows us to pass an immutable set of strings to the hook below in this case.